src/core.ts

```diff
@@ -457,18 +457,9 @@ export function showCompileError(path: string, compile_error: string) {
     let error_path = join(path, ATTIC, "stderr");
     let error_file = openSync(error_path, "w");
     writeSync(error_file, compile_error);
-    vscode.commands.executeCommand("vscode.setEditorLayout", {
-        orientation: 1,
-        groups: [
-            { groups: [{}], size: 0.6 },
-            { groups: [{}], size: 0.4 },
-        ],
-    });
-    vscode.commands.executeCommand(
-        "vscode.open",
-        vscode.Uri.file(error_path),
-        vscode.ViewColumn.Two
-    );
+    let stderr_terminal = vscode.window.createTerminal("Compilation Error");
+    stderr_terminal.show();
+    stderr_terminal.sendText("more " + '"' + error_path + '"');
```

Comment:
`more` Is still a unix based command not available on windows platform, so this will fail on windows machine. 

One workaround to handle this, is test first if the `more` command is available by seeing `more --version` run without error. If it doesn't fail we use it, otherwise use current approach.

Reply:
Maybe we can install `more` in windows automatically (or request to install, if not available). I just checked on [wikipedia](https://en.wikipedia.org/wiki/More_(command)) and it says it is available in Windows, will check.

Reply:
I see it is already available in windows.